features/RST15_audit_events_api.feature

```diff
@@ -0,0 +1,57 @@
+Feature: Creating Audit events is possible from outside the application 
+
+    Background:
+
+        Given fixtures from "bdd_auth" are loaded
+        And fixtures from "bdd_auth" are available
+        And fixtures from "auditevent_valid_exercise_api" are loaded
+	And fixtures from "auditevent_create_api" are available
+        And I am logged into the api interface as "admin"
+        And I am logged into the admin interface as "servicemanager"
+
+    Scenario Outline: A pre-processing step reports failing to process a case
+
+        When an api call is made to create a "audit event"
```

Comment:
I would avoid using 'When an api call '
The only people that are going to know what this means are the devs :(